javascripts/characters.js

```diff
@@ -32,5 +33,27 @@ var Gauntlet = (function(gauntlet) {
     return character;
   };
 
+  gauntlet.Character.randomCharacter = function(name) {
+    let randomName = getRandElem(name);
+    let randomSpecies = getRandElem(gauntlet.Species.getSpeciesList());
+    let allowedClasses = Gauntlet.Species.getSpeciesList().allowedClasses;
```

Comment:
allowedClasses won't work here. The property is on an individual species, not a list.